Cargo.toml

```diff
@@ -155,7 +155,7 @@ libsecp256k1 = "0.7"
 log = "0.4"
 lru = "0.12"
 maplit = "1"
-milhouse = "0.3"
+milhouse = { rev = "1df8db756200fa1ad77af17dbc036b85a444fd1a", git = "https://github.com/paulhauner/milhouse" }
```

Comment:
I think we need a Milhouse 0.4 release when you think the time is right

Reply:
Here we go!

https://github.com/sigp/milhouse/pull/59